src/components/bookList.js

```diff
@@ -1,10 +1,16 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import AddBook from './addBook';
 import Book from './Book';
+import { fetchBook } from '../redux/books/booksSlice';
 
 const BookList = () => {
   const { books } = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchBook());
+  }, []);
```

Comment:
[OPTIONAL] To enforce best practices as suggested by the react team, the dependency array should not be left empty unless there are no dependencies then it is removed. Kindly remove the dependency array or kindly add the dispatch function as a dependency (NOTE: this will be redundant because the dispatch function will never change) to follow best practices.